src/lib/core/components/VariableCoverageTable.tsx

```diff
@@ -59,7 +59,10 @@ export function VariableCoverageTable({
       ? 'VariableCoverageTable'
       : `${containerClassName} VariableCoverageTable`;
 
-  return (
+  return variableSpecs[0].display ||
+    variableSpecs[1].display ||
+    variableSpecs[2].display ||
+    variableSpecs[3].display ? (
```

Comment:
I was getting a runtime error about variableSpecs[3] being undefined.  One solution is below

```suggestion
  return variableSpecs[0]?.display ||
    variableSpecs[1]?.display ||
    variableSpecs[2]?.display ||
    variableSpecs[3]?.display ? (
```

Another might be something like `variableSpecs.some( (spec) => spec.display != null )` (if the test for not null is appropriate)

Reply:
I have continued testing with the `some(...)` option and that seems good.

Could you talk with @moontrip if he's available and talk about widening the legend to the standard 400px width that the variable coverage table and birds eye are using?

Reply:
I noticed with this approach that histograms are assigned `Main` as the `spec.display` and that bar plots are not. This resulted in the table rendering for histograms before selecting any variables.

I'm changing to `variableSpecs.some( (spec) => spec.variable)` to account for this.